Fixed broken internal links in docs for ash_phoenix (#87)
ash-project/ash_phoenix: documentation/tutorials/getting-started-with-ash-and-phoenix.md

There's a few places you can go to learn more about how to use ash:

> - [Read more about how to query the data in your resources.](/docs/module/ash/latest/ash-query)

Reviewer: ```suggestion
- Read more about how to query the data in your resources: `Ash.Query`
```

> - [Dig deeper into actions.](/docs/guides/ash/latest/topics/actions)

Reviewer: ```suggestion
- [Dig deeper into actions.](https://hexdocs.pm/ash/actions.html)
```

- [Study resource relationship management](/docs/guides/ash/latest/topics/managing-relationships)

### Ash Authentication & Ash Authentication Phoenix

See the power Ash can bring to your web app or API. [Get authentication working in minutes](https://hexdocs.pm/ash_authentication_phoenix/getting-started-with-ash-authentication-phoenix.html).

### Add an API (or two)

> Check out the [AshJsonApi](/docs/guides/ash_json_api/latest/tutorials/getting-started-with-json-api) and [AshGraphql](/docs/guides/ash_graphql/latest/tutorials/getting-started-with-graphql) extensions to effortlessly build APIs around your resources.

Reviewer: ```suggestion
Check out the [AshJsonApi](https://hexdocs.pm/ash_json_api/getting-started-with-json-api.html) and [AshGraphql](https://hexdocs.pm/ash_graphql/getting-started-with-graphql.html#content) extensions to effortlessly build APIs around your resources.
```
